sys-apps/ignition/files/0006-internal-prv-cmdline-backport-flatcar-patche.patch

```diff
@@ -1,40 +0,0 @@
-From 6581b0c3359a77c364d9f776557db8b96b4561c6 Mon Sep 17 00:00:00 2001
-From: Mathieu Tortuyaux <mathieu@kinvolk.io>
-Date: Fri, 30 Jul 2021 14:21:31 +0200
-Subject: [PATCH 06/16] internal/prv/cmdline: backport flatcar patche
-
-this patch provides backward compatiblity for various config
-key
-
-Signed-off-by: Mathieu Tortuyaux <mathieu@kinvolk.io>
----
- internal/providers/cmdline/cmdline.go | 6 ++++--
- 1 file changed, 4 insertions(+), 2 deletions(-)
-
-diff --git a/internal/providers/cmdline/cmdline.go b/internal/providers/cmdline/cmdline.go
-index 0068ed66..80b3af95 100644
---- a/internal/providers/cmdline/cmdline.go
-+++ b/internal/providers/cmdline/cmdline.go
-@@ -33,7 +33,9 @@ import (
- )
- 
- const (
--	cmdlineUrlFlag = "ignition.config.url"
-+	cmdlineUrlFlagLegacyCoreOS = "coreos.config.url"
-+	cmdlineUrlFlagLegacy       = "flatcar.config.url"
-+	cmdlineUrlFlag             = "ignition.config.url"
- )
- 
- func FetchConfig(f *resource.Fetcher) (types.Config, report.Report, error) {
-@@ -82,7 +84,7 @@ func parseCmdline(cmdline []byte) (url string) {
- 		parts := strings.SplitN(strings.TrimSpace(arg), "=", 2)
- 		key := parts[0]
- 
--		if key != cmdlineUrlFlag {
-+		if !(key == cmdlineUrlFlagLegacy || key == cmdlineUrlFlagLegacyCoreOS || key == cmdlineUrlFlag) {
```

Comment:
Wait, can you explain why this patch can go?

Reply:
Yeah sure - it's just not used in the ebuild: https://github.com/flatcar/coreos-overlay/blob/main/sys-apps/ignition/ignition-9999.ebuild#L50. (There is a typo in `patche`)
So we can just drop it, the correct one is: 0006-internal-prv-cmdline-backport-flatcar-patch.patch (which is not impacted by the renaming)

Reply:
Ah, ok, that confused me